qutebrowser/completion/models/base.py

```diff
@@ -26,61 +26,48 @@
 from PyQt5.QtCore import Qt
 from PyQt5.QtGui import QStandardItemModel, QStandardItem
 
-from qutebrowser.utils import usertypes
 
-
-Role = usertypes.enum('Role', ['sort', 'userdata'], start=Qt.UserRole,
-                      is_int=True)
-
-
-class BaseCompletionModel(QStandardItemModel):
+class CompletionModel(QStandardItemModel):
 
     """A simple QStandardItemModel adopted for completions.
 
     Used for showing completions later in the CompletionView. Supports setting
     marks and adding new categories/items easily.
 
-    Class Attributes:
-        COLUMN_WIDTHS: The width percentages of the columns used in the
+    Attributes:
+        column_widths: The width percentages of the columns used in the
                         completion view.
-        DUMB_SORT: the dumb sorting used by the model
     """
 
-    COLUMN_WIDTHS = (30, 70, 0)
-    DUMB_SORT = None
-
-    def __init__(self, parent=None):
+    def __init__(self, column_widths=(30, 70, 0), columns_to_filter=None,
+                 delete_cur_item=None, parent=None):
```

Comment:
Should some of those be keyword-only (i.e. with a `*` argument before them, so they can't be used as positional arguments) maybe?